packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/SyncSession.kt

```diff
@@ -38,6 +39,34 @@ import io.realm.Realm
  */
 public interface SyncSession {
 
+    /**
+     * Calling this method will block until all known remote changes have been downloaded and
+     * applied to the Realm or the specified timeout is hit. This will involve network access, so
+     * calling this method should only be done from a non-UI thread.
+     *
+     * @param timeout Maximum amount of time before this method should return.
```

Comment:
you should specify the unit?

Reply:
`timeOut` is a `Duration`, which is a new Kotlin API that captures both value and duration in a quite succinct way, i.e. you can write `10.seconds`